File: OpenEdXMobile/src/main/java/org/edx/mobile/discussion/CommentBody.java
package org.edx.mobile.discussion;


import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

public class CommentBody {
    private String threadId;
    private String rawBody;
    private String parentId;

    public CommentBody(@NonNull String threadId, @NonNull String rawBody, @Nullable String parentId) {
        this.threadId = threadId;
        this.rawBody = rawBody;
        this.parentId = parentId;
    }
}

File: OpenEdXMobile/src/main/java/org/edx/mobile/discussion/CommentBody.kt
package org.edx.mobile.discussion

import com.google.gson.annotations.SerializedName

data class CommentBody(
    @SerializedName("thread_id")
    val threadId: String,

    @SerializedName("raw_body")
    val rawBody: String,

    @SerializedName("parent_id")
    val parentId: String?
)
